test/machinaris/1.0.6/questions.yaml

```diff
@@ -157,3 +159,20 @@ questions:
                 label: "Value"
                 schema:
                   type: string
+  - variable: enableResourceLimits
+    label: "Enable Pod resource limits?"
+    group: "Resource Limits"
+    schema:
+      type: boolean
+  - variable: cpuLimit
+    label: "CPU limit"
+    group: "Resource Limits"
+    schema:
+      type: string
```

Comment:
Do you think we should perhaps validate if specified value is valid using regex https://github.com/truenas/charts/blob/master/docs/schema/string.md ?

Reply:
Both values should now be regex-validated, the memory value comes straight from within the Kubernetes/`k3s` codebase, the CPU value is mine but has been tested with [Regex101](https://regex101.com/r/0PQNCH/1) to make sure it passes and fails on expected values